integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonAddLoadCommand.scala

```diff
@@ -341,22 +350,6 @@ case class CarbonAddLoadCommand(
           carbonTable, segment, partitionSpecOp.orNull, partitionDataFiles.asJava)
       }
 
-    // This event will trigger merge index job, only trigger it if it is carbon file
-    if (isCarbonFormat) {
-      operationContext.setProperty(
-        carbonTable.getTableUniqueName + "_Segment",
-        model.getSegmentId)
-      // when this event is triggered, SI load listener will be called for all the SI tables under
-      // this main table, no need to load the SI tables for add load command, so add this property
-      // to check in SI load event listener to avoid loading to SI.
-      operationContext.setProperty("isAddLoad", "true")
-      val loadTablePreStatusUpdateEvent: LoadTablePreStatusUpdateEvent =
```

Comment:
why this code removed? now merge code not in listener? and isAddLoad and other operation context property also removed

Reply:
I'm directly calling mergeIndex( using `mergeIndexFilesInAddLoadSegment`) here instead of the listener. So, no need of setting `isAddLoadProperty`. And I have removed this property from SI listener also.